packages/server/src/utils/logging.ts

```diff
@@ -30,28 +30,31 @@ addLayout('json', () => {
     };
 });
 
-const log4jConfig = (layout: string): Configuration => ({
-    appenders: {
-        console: { type: 'console' },
-        fileAppender: {
-            type: 'file',
-            filename: logLocation,
-            maxLogSize: 10000000,
-            backups: 5,
-            compress: true,
-            layout: { type: layout, separator: ',' },
+const log4jConfig = (layout: string): Configuration => {
+    return {
+        appenders: {
+            console: { type: 'console' },
+            fileAppender: {
+                type: 'file',
+                filename: logLocation,
+                maxLogSize: 10000000,
+                backups: 5,
+                compress: true,
+                layout: { type: layout, separator: ',' },
+            },
         },
-    },
-    categories: {
-        default: { appenders: ['fileAppender'], level: 'info' },
-        production: { appenders: ['fileAppender'], level: 'info' },
-        development: { appenders: ['console'], level: 'info' },
```

Comment:
I've removed the prod/dev categories here because even if the `development` category is used, the `production` one still gets run and it tries to create the file in `/var/log/`, which fails locally.

The new logic below means it only writes to file if `NODE_ENV == production`, which covers CODE/PROD (i.e. on an ec2)